packages/theme-toggler-extension/src/index.tsx

```diff
@@ -102,12 +45,11 @@ const extension: JupyterFrontEndPlugin<void> = {
     if (toolbarRegistry) {
       toolbarRegistry.addFactory('TopBar', 'theme-toggler', () => {
         const widget = ReactWidget.create(
-          <ThemeSwitch
-            themeManager={themeManager}
-            onChange={onChange}
-            innerLabel="light"
-            innerLabelChecked="dark"
-          />
+          <Switch onChange={onChange} checked={true}>
```

Comment:
Should `checked` always be true?

Testing on Binder, the theme seems to be changed after a page refresh:

[theme-toggler.webm](https://github.com/user-attachments/assets/5e63be33-7aeb-4f35-bde7-64fa73bd1777)

Reply:
Yes, you are correct. I over simplified the logic. Now I added the logic back to keep the state of the switch based on current theme.